resources/dicts/relationship_events/normal_interactions/trust/increase.json

```diff
@@ -1,349 +1,400 @@
+		],
+		"relationship_constraint": [
+			"trust_30"
+		],
+		"reaction_random_cat": {
+			"trust": "increase"
+		}
+	},
+	{
+		"id": "trust_inc_high4",
+		"intensity": "high",
+		"interactions": [
+			"m_c goes to r_c for advice about some recent troubles.",
+			"m_c asks r_c to help {PRONOUN/m_c/object} find the right words to say to a Clanmate.",
+			"m_c asks r_c for help resolving an argument with a friend."
+		],
+		"relationship_constraint": [
+			"trust_30"
+		],
+		"random_status_constraint": [
+			"elder",
+			"mediator",
+			"mediator apprentice"
+		]
+	},
+	{
+		"id": "trust_star_inc_high1",
+		"intensity": "high",
+		"interactions": [
+			"m_c tells r_c about {PRONOUN/m_c/poss} last encounter with StarClan.",
+			"m_c debates about the meaning of a prophetic dream with r_c.",
+			"r_c puts a lot of faith into m_c's skill as {PRONOUN/m_c/poss} rank."
+		],
+		"main_status_constraint": [
+			"medicine cat apprentice",
+			"medicine cat",
+			"leader"
+		],
+		"relationship_constraint": [
+			"trust_30"
+		],
+		"random_status_constraint": [
+			"elder",
+			"warrior",
+			"mediator",
+			"medicine cat apprentice",
+			"medicine cat",
+			"deputy",
+			"leader"
+		],
+		"reaction_random_cat": {
+			"trust": "increase"
+		}
+	},
+	{
+		"id": "trust_led_inc_high1",
+		"interactions": [
+			"m_c talks with r_c about difficult decisions.",
+			"m_c makes battle plans with r_c.",
+			"m_c discusses future plans for the Clan with r_c.",
+			"m_c catches up on the current Clan relationships with r_c.",
+			"m_c discusses plans to expand the territory with r_c."
+		],
+		"relationship_constraint": [
+			"trust_50"
+		],
+		"main_status_constraint": [
+			"leader"
+		],
+		"random_status_constraint": [
+			"deputy",
+			"elder",
+			"medicine cat"
+		],
+		"reaction_random_cat": {
+			"trust": "increase"
+		}
+	},
+	{
+		"id": "trust_med_inc_high1",
+		"interactions": [
+			"m_c tells r_c a trick about a certain herb.",
+			"m_c shows r_c how to make an herb taste better.",
+			"m_c and r_c invent a new treatment method together.",
+			"m_c knows that r_c will always bring back the right herbs from a patrol.",
+			"m_c trusts r_c with a patient {PRONOUN/m_c/subject} {VERB/m_c/were/was} treating.",
+			"m_c goes to r_c to ask for {PRONOUN/r_c/poss} wisdom.",
+			"m_c asks r_c about a specific treatment method."
+		],
+		"main_status_constraint": [
+			"medicine cat apprentice",
+			"medicine cat"
+		],
+		"random_status_constraint": [
+			"medicine cat apprentice",
+			"medicine cat"
+		],
+		"reaction_random_cat": {
+			"trust": "increase"
+		}
+	},
+	{
+		"id": "trust_med_inc_high2",
+		"interactions": [
+			"m_c observes r_c carefully attending to any cat that needs help.",
+			"m_c feels like {PRONOUN/m_c/subject} can count on r_c to give {PRONOUN/m_c/object} the best treatment possible."
+		],
+		"random_status_constraint": [
+			"medicine cat apprentice",
+			"medicine cat"
+		]
+	}
+]
```

Comment:
same as the prior file— would like to know the reasoning for removed interactions :]